tests/monitoring_test.go

```diff
@@ -149,12 +235,11 @@ var _ = Describe("[Destructive] Monitoring Tests", func() {
 				Expect(err).ToNot(HaveOccurred())
 			}
 
-			expectedIncomplete := originalMetricVal + numAddedStorageClasses
+			expectedIncomplete := numAddedStorageClasses
```

Comment:
how come you're able to get rid of this?

Reply:
I'll re-add it. Not sure why would we encounter incomplete StorageProfiles in our CI, and more specificly during these destructive tests. But of course when you run it on a random cluster it may happen.

Reply:
Isn't "local" incomplete?

Reply:
It has `provisioner: kubernetes.io/no-provisioner` annotation so we use `capabilitiesForNoProvisioner()` as mentioned in your previous comment answer.